PipelineCommon/Helpers/MarkdigExtensions/RCLinkRenderer.cs

```diff
@@ -94,6 +95,29 @@ protected override void Write(HtmlRenderer renderer, RCLink obj)
                 return;
             }
 
+            match = BIBLE_LINK.Match(rcLinkText);
+            if (match.Success)
+            {
+                var language = match.Groups[1].Value;
+                if (language == "*")
+                {
+                    language = _options.LanguageCode;
+                }
+
+                var bibleVersion = match.Groups[2];
+                var book = match.Groups[3];
+                var chapter = match.Groups[4];
+                var verse = match.Groups[5];
+
+                if (_options.RenderAsBTTWriterLinks)
+                {
+                    RenderBTTWriterLink(renderer, $":{language}:bible:{bibleVersion}:{book}:{chapter}:{verse}|");// The | is nessecary to match the regex in writer. What does it do? Absolutely nothing.
+                    return;
+                }
+                RenderLink(renderer, $"{_options.ServerUrl}/u/{_options.BaseUser}/{language}_{bibleVersion}/{book}.html#chp-{chapter}-vs-{verse}");

```

Comment:
Should there be an else here?  Or should `RenderLink()` be called even if `RenderAsBTTWriterLinks` is true?

Reply:
Oh, of course!  I missed that.  :)